openfecli/parameters/output_dir.py

```diff
@@ -0,0 +1,22 @@
+import click
+import pathlib
+from plugcli.params import MultiStrategyGetter, Option, NOT_PARSED
+
+
+def get_dir(user_input, context):
+    dir_path = user_input
+    return dir_path
+
+
+def ensure_file_does_not_exist(ctx, param, value):
+    if value and value.exists():
+        raise click.BadParameter(f"File '{value}' already exists.")
+    return value
```

Comment:
this function is not used and is not tested; should it be removed?

Reply:
jap